zerver/models.py

```diff
@@ -3313,6 +3313,23 @@ def increment_times_used(self) -> None:
         self.save(update_fields=["times_used"])
 
 
+class MissedMessageEmailEntry(models.Model):
```

Comment:
What do you think about calling this `ScheduledMissedMessageEmailTrigger`?  I think that might convey the intention a bit more clearly, with analogy to `ScheduledEmail` and `ScheduledMessage`.

Reply:
(I think there's a good chance that we're going to rename `MissedMessage` across the codebase, since we changed the user-facing name for the email sender from "Zulip missed messages" to "Zulip [email is implicit] notifications", and we plan to have configurations that send these even if you see the message)

Reply:
`ScheduledMessageNotificationEmail` sounds good, updated to use that 👍